src/flexalloc_mm.c

```diff
@@ -727,38 +708,74 @@ fla_base_sync(struct flexalloc *fs)
 }
 
 void
-fla_print_pool_entry(struct flexalloc *fs, struct fla_pool *pool)
+fla_print_geo(struct flexalloc *fs)
+{
+  struct fla_geo * geo = &(fs->geo);
+  fprintf(stderr, "Flexalloc Geometry: \n");
+  fprintf(stderr, "|  LBAs: %"PRIu64"\n", geo->nlb);
+  fprintf(stderr, "|  LBA width: %"PRIu32"B\n", geo->lb_nbytes);
+  fprintf(stderr, "|  md blocks: %"PRIu32"\n", geo->md_nlb);
+  fprintf(stderr, "|  pool segment:\n");
+  fprintf(stderr, "|    * pools: %"PRIu32"\n", geo->npools);
+  fprintf(stderr, "|    * freelist blocks: %"PRIu32"\n", geo->pool_sgmt.freelist_nlb);
+  fprintf(stderr, "|    * htbl blocks: %"PRIu32"\n", geo->pool_sgmt.htbl_nlb);
+  fprintf(stderr, "|    * entry blocks: %"PRIu32"\n", geo->pool_sgmt.entries_nlb);
+  fprintf(stderr, "|    * total blocks: %"PRIu32"\n", fla_geo_pool_sgmt_nblocks(&geo->pool_sgmt));
+  fprintf(stderr, "|  slab segment:\n");
+  fprintf(stderr, "|    * slabs: %"PRIu32"\n", geo->nslabs);
+  fprintf(stderr, "|    * total blocks: %"PRIu32"\n", geo->slab_sgmt.slab_sgmt_nlb);
```

Comment:
I would add slab total blocks to the print here, just total blocks can be ambiguous

Reply:
done